posthog/utils.py

```diff
@@ -413,3 +413,73 @@ def get_redis_info() -> Mapping[str, Any]:
 
 def get_redis_queue_depth() -> int:
     return get_client().llen("celery")
+
+
+from collections import defaultdict
```

Comment:
Suggestion: Let's extract this under `posthog/queries/__init__.py` - rather than utils these are query-specific tools.